aml-development/frontend
# feat(management): table view improvements

AMLOS-765

**Description**
Notes from PR (https://github.com/aml-development/frontend/pull/142)

**Acceptance Criteria**
- maybe the Row # in the table view (1,2,3, etc), and/or a count next to the Prev/Next buttons (1-24, 25-50 - or 25-44 if a lesser number)
- need the status icon for a Listing while in Listing management. We might be able to get away with keeping the overall ListingTile component, just with that visual indicator. (tracked by AMLOS-775)
- 2.0 remembers what you had the view set to (table or tile) on page reload
- 2.0 allows you to search the table
- 2.0 allows you to remove columns you dont want to see and saves it to a cookie.

**How to Test**
Pull this branch

- Check row number (#) column in any table view
- Check row count at bottom of any table view
- Verify that app loads grid vs table view based on session (defaults to grid)
- Verify functionality of column selector (button to the left of search bar in any table view)
- Verify functionality of table search in any table view
- Verify that app loads visible/hidden columns based on session (defaults to all visible)
